calico_upgrade/pkg/migrate/felixconfig_test.go

```diff
@@ -26,10 +29,12 @@ import (
 
 var _ = Describe("Test felix configuration upgrade", func() {
 	// Define some common values
-	int1 := int(12345)
+	seconds1 := metav1.Duration{Duration: time.Duration(12345) * time.Second}
+	millis1 := metav1.Duration{Duration: time.Duration(12345) * time.Millisecond}
```

Comment:
nit:  Can we use different values for both of these (same below).  It makes it easier when reading trace.

Reply:
They technically are (seconds vs ms) but yea I'll make them more differenter